WordPress/wordpress-develop
# Update terser webpack plugin

Trac ticket: https://core.trac.wordpress.org/ticket/58660

**TESTING INSTRUCTIONS**

After applying this PR, run a new build locally by running `npm run build` (or `npm run build:dev` if you are running WP from `src`). Make sure `SCRIPT_DEBUG` is set to `false` in your `wp-config.php`, and load the block editor in the admin. It should load properly and no longer white screen.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**
